cli/spec.py

```diff
@@ -0,0 +1,244 @@
+import ConfigParser
+import os
+import sys
+import yaml
+
+import clg
+import yamlordereddictloader
+from cli import exceptions
+from cli import utils
+
+SPEC_EXTENSION = '.spec'
+LOOKUP_DICT = {
+    '__DEFAULT__': "default"
+}
+
+TRIM_PARAMS = ['default', 'required', 'requires_only']
+
+
+def clg_ini_file(value):
```

Comment:
Function's name should describes what it does. In this case, this is function isn't related to 'clg'.
I suggest something like "ini_to_dict" "cfg_file_to_dict".